docs/development/issue-labeling.md

```diff
@@ -36,7 +36,6 @@ Most issues should have a label relating to either a platform, manager, datasour
     status:blocked
     status:ready
     status:in-progress
-    status:waiting-on-response
```

Comment:
Line 44 still references the removed `status:waiting-on-response` label. 😉 

> If we need the original poster or somebody else to respond to a query of ours, apply the `status:waiting-on-response` label.

The search query string on line 45 needs to be updated as well. 😉